build.sh

```diff
@@ -0,0 +1,19 @@
+# NOTE: kolibri whl file must be located in the current directory
+
+mkdir -p bin
+rm bin/*.apk 2> /dev/null
+
+docker build -t kolibrikivy .
```

Comment:
These 2 docker commands can just be added to the makefile? The reason I added this script was because there `<< COMMANDS` was not working within the makefile. But since you removed this, we can do without this script.